samza-rest/src/main/java/org/apache/samza/monitor/SamzaMonitorService.java

```diff
@@ -62,9 +64,16 @@ public void start() {
 
                 if (!Strings.isNullOrEmpty(monitorConfig.getMonitorFactoryClass())) {
                     int schedulingIntervalInMs = monitorConfig.getSchedulingIntervalInMs();
-                    LOGGER.info("Scheduling monitor {} to run every {} ms", monitorName, schedulingIntervalInMs);
                     // MetricsRegistry has been added in the Monitor interface, since it's required in the eventual future to record metrics.
                     // We have plans to record metrics, hence adding this as a placeholder. We just aren't doing it yet.
+
+                    /**
+                     *  Jitter is generated by multiplying a random number from the range([0, schedulingIntervalInMs]) with
```

Comment:
Can remove comment, does not add anything to what the next line already describes.

Reply:
Removed the comment.